text/0010-gatsby-docs-localization

```diff
@@ -0,0 +1,104 @@
+
+For larger websites, especially documentation websites that have lots of PRs and require versioning and up-to-date content, this isn't enough. The docs for [Angular](https://github.com/angular/angular-cn/tree/aio), [Vue](https://github.com/vuejs/jp.vuejs.org), and [React](https://github.com/reactjs/es.reactjs.org) store their translations in copies of the repo.
+
+### Advantages
+
+This was the approach that we took for translating the React docs. It appealed to us for a lot of reasons:
+
+* Open Source Contributors tend to be familiar with Git
+* No need to integrate with an external API and software (aside from GitHub)
+* People want to be listed as contributors of the React/Gatsby organization on GitHub, and it's an incentive to contribute.
+* Can use GitHub features such as Code Owners and pull requests as quality assurance
+
+### Drawbacks
+
+The main drawback of this approach is that the Gatsby docs currently live in a monorepo with the rest of the Gatsby source, which means we can't just do a copy of the entire website and use `git merge` like we do with React (and Vue).
+
+This is also an issue if we want translations to live under a path (e.g. `gatsbyjs.org/ja` rather than `ja.gatsbyjs.org`).
+
+Other drawbacks we learned from working on the React translation:
+
+* Need to maintain a custom bot and system to keep the translations up to date.
+* It's not a system that can be easily put into a plugin to allow others to use.
+
+### Mitigation
+
+I think this is a viable approach if we can:
+
+1. move the docs to their own repo and
+2. are okay with doing subdomains (`ja.gatsbyjs.org`) instead of paths (`gatsbyjs.org/ja`)
+
+Otherwise we might want to consider the second alternatives:
+
+## Alternative 2: Translations Platform
+
+[Crowdin]() is a SaaS that allows projects to source translations using a built-in web based editor. It's popular in the JS community being used for projects such as [Electron](https://crowdin.com/project/electron) and [Yarn](https://crowdin.com/project/yarn).
+
+### Advantages
+
+* Lots of features, such as [machine translation](https://support.crowdin.com/pre-translation-via-machine/) and [in-context localization](https://support.crowdin.com/in-context-localization/).
+* Has a builtin [GitHub integration](https://support.crowdin.com/github-integration/).
+* We can make an official plugin for Crowdin integration that other projects can use.
+
+### Drawbacks
+
+We had considered using Crowdin for the localization of the React docs but realized that Crowdin had some [major drawbacks](https://reactjs.org/blog/2019/02/23/is-react-translated-yet.html):
+
+* It has a steep learning curve and hard for new translators to get accustomed to.
+* The GitHub integration isn't customizable: it takes three hours to compile all the languages and just publishes something in a single directory.
```

Comment:
Should be handled by our build process, so I don't see any issues here

Reply:
From the people who I've talked to that have actually done or attempted to do the integration, it's a huge pain. If you have experience with this and we decide to go with crowdin, please feel free to help.